composer.json

```diff
@@ -6,27 +6,28 @@
   "description": "Encrypted symfony entity's by verified and standardized libraries",
   "require": {
     "php": "^7.2",
-    "paragonie/halite": "^4.3",
-    "paragonie/sodium_compat": "^1.5",
```

Comment:
people who relied on this bundle to make sodium compat installed will no longer get it. transitive dependencies are fragile, but this might trip up people.

Reply:
No problem, best to take the lowest risk approach. I've reverted.